charts/islandis/values.dev.yaml

```
@@ -2227,6 +2227,7 @@ services-sessions:
     DB_NAME: 'services_sessions'
     DB_REPLICAS_HOST: 'dev-vidspyrna-aurora.cluster-ro-c6cxecmrvlpq.eu-west-1.rds.amazonaws.com'
     DB_USER: 'services_sessions_read'
+    GEODATADIR: '/geoip-lite/data'
```

Consider changing this to what it effects. If the data is `lite`, it should say so in the filename. This way you could add `hevy` data in same folder.
```suggestion
    GEODATADIR: '/geoip/data'
```

---

the library is called geoip-lite, not geoip.